tests/test_url_parsing.py

```diff
@@ -239,8 +239,13 @@ def test_no_path(self):
         assert u.fragment == ""
 
     def test_no_host(self):
```

Comment:
Can you please mark the test as 'expected to fail'?

```suggestion
    @pytest.mark.xfail
    def test_no_host(self):
```